### DISCUSSION TITLE
Filter the field that have relationship

### DISCUSSION THREAD

**Discussion by EronitaB** [Budibase/budibase#2447] [Help]
Hi there,
I am working on this project and I want to add a filter here in order to search by the name of the client (klienti).
Momentally I can't make it happen, is there a way to filter the data, **even that they are taken by other tables**(they have relationship)?
Thanks.
<img width="1608" alt="Screen Shot 2021-08-24 at 14 05 11" src="https://user-images.githubusercontent.com/77198959/130613444-6ead32f1-cf6c-44bc-9df8-ee9727ab6dd0.png">

**Comment by mjashanks** [comment]
Hi @EronitaB 

This documentation describes how to do filtering: https://docs.budibase.com/design/handling-data/filtering-data

However, you cannot filter by data in a **related** table at the moment. Right now, this would only be possible by using a SQL Datasource.
